test/controllers/users_controller_test.rb

```diff
@@ -185,6 +185,31 @@ class UsersControllerTest < ActionDispatch::IntegrationTest
     end
   end
 
+  describe 'clear_call_list method' do
+    before do
+      patch add_patient_path(@user, @patient_1), xhr: true
+      patch add_patient_path(@user, @patient_2), xhr: true
+      patch clear_call_list_path(@user), xhr: true
```

Comment:
Can you add some kind of check that our user has 2 patients here? Maybe an assert or raise something?